GhostText/Services/MessageService.cs

```diff
@@ -29,13 +30,13 @@ public IQueryable<Message> RetrieveAllMessages()
         public async Task<Message> RetrieveMessageByIdAsync(Guid messageId)
         {
             var message = await this.messageRepository.SelectMessageByIdAsync(messageId);
-
-            if (message is null)
+            if (message == null)
```

Comment:
```suggestion
            if (message is null)
```

`is` dan foydalaning == o'rniga.